clients/imodelhub/src/imodelbank/IModelBankFileSystemITwinClient.ts

```diff
@@ -66,19 +62,15 @@ export class IModelBankFileSystemITwinClient implements ITwinManagerClient {
     return props.map((value) => value.properties as IModelFileSystemContextProps);
   }
 
-  // SWB
   public async getITwinByName(requestContext: AuthorizedClientRequestContext, name: string): Promise<ITwin> {
-    const props = await this.queryContextProps(requestContext, name); // SWB what does context mean here
-    requestContext.enter();
+    const props = await this.queryContextProps(requestContext, name);
```

Comment:
Does the rename apply to this use of context?

Reply:
Removed questions from clients since they will be removed soon. Marking this as resolved.